src/TripLogs.js

```diff
@@ -464,6 +464,13 @@ class TripLogs {
           if (newTripId === undefined) {
             nonTripIdx++;
           }
+          const plannedPath = _.get(
+            le,
+            "response.remainingvehiclejourneysegments[0].path"
+          );
+          if (plannedPath && plannedPath.length > 0) {
```

Comment:
yes this only shows the first planned path. in the future we could make it so that it would show different planned paths. would that be based on the left end of the slider, or something else?